BI_Tools/Functions/fastq_filter_funcs.py

```diff
@@ -0,0 +1,40 @@
+def get_gc_score(seq: str) -> float:
+    '''
+    Function gc_score, counts GC bound of sequence
+
+    Args: str
+
+    Returns: float
+    '''
+    gc_score = (seq.lower().count("g") + seq.lower().count("c"))*100/len(seq)
```

Comment:
можно сразу строку сделать в нижнем регистре и потом посчитать гс состав. как вариант)
```suggestion
    seq = seq.lower()
```